install/kubernetes/helm/istio/charts/galley/templates/clusterrole.yaml

```diff
@@ -14,18 +14,33 @@ rules:
 - apiGroups: ["config.istio.io"] # istio mixer CRD watcher
   resources: ["*"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["config.istio.io"]
+  resources: ["*/status"]
+  verbs: ["update"]
 - apiGroups: ["networking.istio.io"]
   resources: ["*"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["networking.istio.io"]
+  resources: ["*/status"]
+  verbs: ["update"]
 - apiGroups: ["authentication.istio.io"]
   resources: ["*"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["authentication.istio.io"]
+  resources: ["*/status"]
+  verbs: ["update"]
 - apiGroups: ["rbac.istio.io"]
   resources: ["*"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["rbac.istio.io"]
+  resources: ["*/status"]
+  verbs: ["update"]
 - apiGroups: ["security.istio.io"]
   resources: ["*"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["security.istio.io"]
+  resources: ["*/status"]
+  verbs: ["update"]
```

Comment:
FYI, these can be grouped to reduce duplication.

apiGroups: ["security.istio.io", "networking.istio.io", "rbac.istio.io"] # etc
resources: ["*/status"]
verbs: ["update"]

Reply:
At least I've verified that works for the get/list/watch verbs.

Reply:
Thanks, I coalesced them together, both for watch etc and status.